Makefile

```diff
@@ -61,11 +62,27 @@ container-uninstall:
 	sudo hack/update_binaries.sh delete
 
 .PHONY: godep
+ifeq ($(GO111MODULE),on)
+godep:
+	$(GO_BIN) build -v ./...
+	make tidy
+else
 godep:
 	dep ensure
+endif
+
+update:
+	$(GO_BIN) get -u
+
+tidy:
```

Comment:
if it's only used in the case of `GO111MODULE`, let's move this into the `if` block above.

Reply:
I'll double check might have been messed up with my rebase.